src/FriendPanel.css

```
@@ -0,0 +1,31 @@
+.FriendsPanel {
```

```suggestion
.friends-panel {
```
Even though we use a css-in-js solution called `styled-components` (https://styled-components.com/ if you're wondering) in most of our codebases in Cognite, `kebab-case` is the most common naming convention for css class names. Not going to add a comment for every class name about this, but just wanted to mention it.